bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/discovery/YeelightDiscoveryService.java

```diff
@@ -118,6 +120,8 @@ private ThingTypeUID getThingTypeUID(DeviceBase device) {
                 return YeelightBindingConstants.THING_TYPE_CEILING4;
             case color:
                 return YeelightBindingConstants.THING_TYPE_WONDER;
+            case color4:
+                return YeelightBindingConstants.THING_TYPE_WONDER;
```

Comment:
This return is redundant since it is the same for color and color4. Change it to:
```
case color:
case color4:
    return YeelightBindingConstants.THING_TYPE_WONDER;
```